File: Anul II/Medii de proiectare si programare/Laborator/lab06/networking/src/main/java/com/ubb/protocol/request/LoginRequest.java
package com.ubb.protocol.request;

public class LoginRequest implements Request {
    private final String username;
    private final String passwordToken;

    public LoginRequest(String username, String passwordToken) {
        this.username = username;
        this.passwordToken = passwordToken;
    }

    public String getUsername() {
        return username;
    }

    public String getToken() {
        return passwordToken;
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab08/C#/Ubb.BikeContest.Networking/Protocol/Request/LoginRequest.cs
using System;

namespace Ubb.BikeContest.Networking.Protocol.Request
{

    [Serializable]
    public class LoginRequest : IRequest
    {
        private readonly string username;
        private readonly string passwordToken;

        public LoginRequest(string username, string passwordToken)
        {
            this.username = username;
            this.passwordToken = passwordToken;
        }

        public virtual string Username
        {
            get { return username; }
        }

        public virtual string Token
        {
            get { return passwordToken; }
        }
    }

}